File: JavaScript/BilanganGanjilGenap.js
const prompt = require("prompt-sync")({ sigint: true });
let bilangan = parseInt(prompt("Masukkan Bilangan: "));

if (bilangan % 2 == 0) {
  console.log(bilangan + " merupakan bilangan genap.");
  //   console.log(`${bilangan} merupakan bilangan genap.`);
} else {
  console.log(bilangan + " merupakan bilangan ganjil.");
  //   console.log(`${bilangan} merupakan bilangan ganjil.`);
}

File: C++/BilanganGanjilGenap.cpp
#include <iostream>

using namespace std;

int main()
{
    // deklarasi variabel
    int bilangan;

    cout << "Masukkan bilangan: ";
    cin >> bilangan;

    // proses
    if (bilangan % 2 == 0)
    {
        cout << bilangan << " merupakan bilangan genap.\n";
    }
    else
    {
        cout << bilangan << " merupakan bilangan ganjil.\n";
    }

    // memberhentikan program
    return 0;
}